gst/nnstreamer/tensor_common.h

```diff
@@ -62,6 +62,7 @@ typedef enum
   SYNC_NOSYNC = 0,
   SYNC_SLOWEST = 1,
   SYNC_BASEPAD = 2,
+  SYNC_REFRESH = 3,
```

Comment:
Define and elaborate what is REFRESH, exposed by Doxygen entries.

Reply:
OK, I will

Reply:
fortunately, we already have [the document ](https://github.com/nnstreamer/nnstreamer/wiki/Synchronization-Policies-at-Mux-and-Merge)which describes the time sync modes. So, I added a @see tag at the enum `tensor_time_sync_mode`.